hal/armv7m/cpu.c

```diff
@@ -199,7 +203,39 @@ char *hal_cpuInfo(char *info)
 
 char *hal_cpuFeatures(char *features, unsigned int len)
 {
-	features[0] = '\0';
+	unsigned int n = 0;
+#ifdef CPU_IMXRT
+	if ((len - n) > 5) {
+		hal_strcpy(features + n, "FPU, ");
+		n += 5;
+	}
+
+	if ((len - n) > 9) {
+		hal_strcpy(features + n, "MPU 16R, ");
```

Comment:
Number of MPU regions should be extracted from MPU_RNR register (https://developer.arm.com/documentation/ddi0439/b/Memory-Protection-Unit/MPU-programmers-model)

Reply:
FYI, In the current plo (not feature/syspage), at this stage the plo has already detected the `mpu` regions, information about which is passed through syspage to the kernel, so it can be reused, also additional information about pre-allocated regions can be printed there (allocated/max).

Reply:
Currently, the best approach is to omit the the number of `MPU` regions and add it in the future when `MPU` interface will be implemented in the kernel. I don't want to add one function only to get MPU region.